app/forms/waste_exemptions_engine/contact_address_lookup_form.rb

```diff
@@ -10,10 +10,9 @@ class ContactAddressLookupForm < AddressLookupFormBase
     validates :contact_address, "waste_exemptions_engine/address": true
 
     def submit(params)
-      # Assign the params for validation and pass them to the BaseForm method for updating
-      contact_address = create_address(params[:contact_address][:uprn], :contact)
+      contact_address_attributes = get_address_data(params[:contact_address][:uprn], :contact)
```

Comment:
Since those forms are not actual `active_record` objects, the view is generating the names of this params using `contact_address` rather than `contact_address_attributes`. 
Since we have to populate those parameters anyway, and the code required for the form to behave like a model with the correct `accept_nested_attributes` defined methods is a long hack of delegation, I have decided to keep the params as they are now, and to rename the key to the correctly scoped `_attributes` when calling `super`. 
Note that the name of the key is important for the mass-assignment on the transient_registration to work.   
This trick is applied to other forms as well in this PR for the same reasons.